test/unit/org/apache/cassandra/db/CoordinatorLogBoundariesTest.java

```diff
@@ -1,137 +0,0 @@
-/*
```

Comment:
This should use `git mv` to preserve the history

Reply:
These did end up as renames in 0162edc101cd03e7aa1ea12d6dd7f3ccedb7c8dc:
```
 test/unit/org/apache/cassandra/{db/CoordinatorLogBoundariesTest.java => replication/CoordinatorLogOffsetsTest.java}       | 42 ++++++++++++++++++++----------------------
```

I think they progressively exceeded the rename threshold after a couple more commits, so the PR diff shows them as rm + add.